frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -81,8 +81,10 @@ Feature: Kiali Overview page
     And the "healthy" application indicator should list the application
 
   @sleep-app
+  @error-rates-app
```

Comment:
@mkralik3 checking against https://github.com/kiali/kiali/pull/7625/files there is bookinfo-app and not error-rates-app

Reply:
oh I see error-rates-app in master also. I guess than its fine!

Reply:
fixed